specs/daito/OpenAPI_Design.md

```diff
@@ -0,0 +1,164 @@
+# VMware Ecosystem OpenAPI
+
+**Author(s)**: [zengyingzhe](https://github.com/zengyingzhe)
+
+## Summary
+
+This documentation serves as the design spec for VMware OpenAPI.
+VMware OpenAPI is a framework which provides generic storage related APIs to upper VMware plugins.
```

Comment:
Plugins are used in many layers.  It is not clear to me what these upper VMware plugins are.  Are they provided by VMware, a third party, or something else.  What is the purpose of this upper plugin?

It will be helpful to create a table with terminologies, explaining exactly what this upper VMware plugin is with an example, and also explaining the meaning of other plugins you mentioned later in the doc.